deploy-deps.sh

```diff
@@ -3,12 +3,48 @@
 script_path="$(dirname -- "${BASH_SOURCE[0]}")"
 
 main() {
+    echo "Checking requirements" >&2
+    check_req
     echo "Deploying Konflux Dependencies" >&2
     deploy
     echo "Waiting for the dependencies to be ready" >&2
     "${script_path}/wait-for-all.sh"
 }
 
+check_req(){
+    # declare the requirements
+    requirements=(kubectl openssl)
+    uninstalled_requirements=()
```

Comment:
please add local to the array declaration

Reply:
Done